src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import classnames from 'classnames';
+
+function getTab(tabsList, currentId) {
+  return tabsList.find(tab => tab.id === currentId) || tabsList[0];
+}
```

Comment:
wll be good in future move this function into separated file, like `helpers` or `utils`